src/Aspire.Cli/Backchannel/ExtensionRpcTarget.cs

```diff
@@ -12,15 +12,22 @@ namespace Aspire.Cli.Backchannel;
 
 internal interface IExtensionRpcTarget
 {
+    Func<string, ValidationResult>? ValidationFunction { get; set; }
+
+    [JsonRpcMethod("getCliVersion")]
```

Comment:
I don't know when this started failing, but JsonRpcMethodAttribute needs to be moved to the interface, not the impl

Reply:
Well, that makes much more sense. Reverted thanks